### PR TITLE
Convert tests to blobstore

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+VERSION 0.2.0 (Released TDB)
+----------------------------
+
+BACKWARD INCOMPATIBILIES:
+
+- Changes to make the client fully compatible with the Blobstore
+  - getFormat() has been moved from ShockFileInformation to ShockNode.
```
Comment (MrCreosote) [inline RELEASE_NOTES.md:7]: Sooo... it turns out that the blobstore has had the `format` key in the wrong place for 5+ years and nobody noticed. Oops.

File: src/us/kbase/shock/client/test/ShockTests.java [hunk 2]
```diff
@@ -669,36 +679,22 @@ private void testFile(String content, String name, String format, ShockNode sn)
...
 	public void invalidFileRequest() throws Exception {
-		ShockNode sn = BSC1.addNode(new ByteArrayInputStream("".getBytes()), 0, "f", null);
-		try {
-			sn.getFile(new ByteArrayOutputStream());
-			fail("Got file from node w/o file");
-		} catch (ShockNoFileException snfe) {
-			assertThat("no file exc string incorrect", snfe.getLocalizedMessage(), 
-					is("Node has no file"));
-		}
+		ShockNode sn = BSC1.addNode(new ByteArrayInputStream("a".getBytes()), 1, "f", null);
```
Comment (MrCreosote) [inline src/us/kbase/shock/client/test/ShockTests.java:690]: You can't store an empty file in the blobstore so all these tests that throw exceptions on empty nodes are impossible to run

File: src/us/kbase/shock/client/ShockFileInformation.java [hunk 3]
```diff
@@ -23,7 +23,6 @@ private ShockFileInformation(){}
 	@JsonProperty("checksum")
 	private Map<String, String> checksum;
```
Comment (Xiangs18) [inline src/us/kbase/shock/client/ShockFileInformation.java:24]: do we need @JsonProperty("checksum") here?
Is not mapping to a diff name though.

Comment (MrCreosote) [reply src/us/kbase/shock/client/ShockFileInformation.java:24]: Probably not, but if I fix all the weird problems with this code this PR will explode, so we should limit the scope to problems with the PR specifically. 

Really there's not much need for this client any more other than backwards compatibility, it's easy enough to do most operations via a generic REST client. I don't see much of a future for it long term, but for now we need it

**Review by Xiangs18 — APPROVED** [review]
LGTM!

**Pull request merged**